AI/VCAI/AIUtility.cpp

```diff
@@ -375,33 +375,6 @@ int3 whereToExplore(HeroPtr h)
 	int radius = h->getSightRadius();
 	int3 hpos = h->visitablePos();
 
-	auto sm = ai->getCachedSectorMap(h);
-
-	//look for nearby objs -> visit them if they're close enouh
-	const int DIST_LIMIT = 3;
-	std::vector<const CGObjectInstance *> nearbyVisitableObjs;
-	for(int x = hpos.x - DIST_LIMIT; x <= hpos.x + DIST_LIMIT; ++x) //get only local objects instead of all possible objects on the map
-	{
-		for(int y = hpos.y - DIST_LIMIT; y <= hpos.y + DIST_LIMIT; ++y)
-		{
-			for(auto obj : cb->getVisitableObjs(int3(x, y, hpos.z), false))
-			{
-				int3 op = obj->visitablePos();
-				CGPath p;
-				ai->myCb->getPathsInfo(h.get())->getPath(p, op);
-				if(p.nodes.size() && p.endPos() == op && p.nodes.size() <= DIST_LIMIT)
-				{
-					if(ai->isGoodForVisit(obj, h, *sm))
-						nearbyVisitableObjs.push_back(obj);
-				}
-			}
-		}
-	}
-	vstd::removeDuplicates(nearbyVisitableObjs); //one object may occupy multiple tiles
-	boost::sort(nearbyVisitableObjs, CDistanceSorter(h.get()));
-	if(nearbyVisitableObjs.size())
-		return nearbyVisitableObjs.back()->visitablePos();
-

```

Comment:
What happened to this code?

Reply:
It is a part of Conquer now. Explore just explores. It moves only 1 tile so if there is something to get we will receive a task to get it. If it has enough priority we will do it instead of exploring. Kind of single responsibility. Sure it eats more resources especially CPU.